http/headers/feature-policy.json

```diff
@@ -868,34 +763,13 @@
             "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Feature-Policy/legacy-image-formats",
             "support": {
               "chrome": {
-                "version_added": "68",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "#enable-experimental-productivity-features",
-                    "value_to_set": "Enabled"
-                  }
-                ]
+                "version_added": false
```

Comment:
I've confirmed that this was dropped. https://bugs.chromium.org/p/chromium/issues/detail?id=874071